cypress/integration/my-orcid/my-orcid-side-bar.e2e.js

```diff
@@ -33,20 +33,127 @@ describe.only('My Orcid sidebar' + runInfo(), () => {
     })
     it('display url to navigate to the public page view', () => {})
   })
-  describe('Emails' + runInfo(), () => {
-    it('display a user with only a primary email "unverified"', () => {})
+  describe.only('Emails' + runInfo(), () => {
+    before(() => {
+      cy.cleanEmails()
+      cy.visit(`${environment.baseUrl}/qa/my-orcid`)
+    })
+    beforeEach(() => {
+      Cypress.Cookies.preserveOnce('XSRF-TOKEN', 'JSESSIONID')
+    })
+    after(() => {
+      cy.cleanEmails()
+    })
+
+    it('display a user with only a primary email "unverified"', () => {
+      cy.get('#emails-panel').within(() => {
+        cy.get('[body=""]').children().should('have.length', 1)
+      })
+    })
     it('display a user with only a primary email "verified"', () => {})
     it('display multiple emails "unverified" and "verified"', () => {
       // Expect changes to be display outside and inside of the modal
     })
     describe('Emails edit modals' + runInfo(), () => {
-      it('add additional emails and display those as "unverified" with default privacy', () => {
-        // Expect changes to be display outside and inside of the modal
+      //TODO we need to figure out how to verify emails to improve testing on those
+
+      it('add additional emails and display those as "unverified" as private', () => {
+        const randomUserEmail = randomUser()
+        cy.get('#emails-panel')
+          .within(() => {
+            cy.get('#edit-button').click()
+          })
+          .get('#modal-container')
+          .get('#add-link')
+          .click()
+          .get('#emailInput-1')
+          .type(randomUserEmail.email)
```

Comment:
@leomendoza123 I had some trouble using the type but it was intermittent. You can implement this https://github.com/ORCID/orcid-angular/blob/63309583aea1512015fec719526228e1d3d0a7fb/cypress/integration/my-orcid/my-orcid-top-bar.e2e.js#L92 to avoid the error

Reply:
The default delay is 10 ms. How is removing this delay completely beneficial? Do you have more info about this, is it a Cypress know bug? 

I have had issues before when I click around the app while the test is running. Is there a possibility that is what was giving you issues?

Reply:
Marking as solve at the moment. Please reopen if you think this change is required now.